### ISSUE TITLE
Install fails on windows due to makedrive dependencies

### ISSUE THREAD

**Issue by k88hudson** [mozilla/webmaker-android#460]
This is an issue @fpanico reported, install is indeed failing on windows due to an issue with maria-sql, a dependency of makedrive.

I'm hitting the issue described here: https://github.com/mscdex/node-mariasql/issues/30
